frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx

```diff
@@ -293,41 +180,50 @@ export const SidePanelSupport = (): JSX.Element => {
-                                                onClick={() => {
-                                                    openSidePanel(
-                                                        SidePanelTab.Docs,
-                                                        '/docs/new-to-posthog/understand-posthog?chat=open'
-                                                    )
-                                                }}
-                                                targetBlank={false}
-                                                className="mt-2"
-                                            >
-                                                🦔 Chat with Max AI
-                                            </LemonButton>
-                                        </>
-                                    </Section>
-                                </FlaggedFeature>
+                                <Section title="Ask Max AI">
+                                    <div>
+                                        <p>Max AI can now answer 80%+ of the support questions we receive! Nice.</p>
+                                        <p>
+                                            Let Max read 100s of pages of docs for you, write SQL queries and
+                                            expressions, regex patterns, etc.
+                                        </p>
+                                        <LemonButton
+                                            type="primary"
+                                            fullWidth
+                                            center
+                                            onClick={() => {
+                                                openSidePanel(
+                                                    SidePanelTab.Docs,
+                                                    '/docs/new-to-posthog/understand-posthog?chat=open'
+                                                )
+                                            }}
+                                            targetBlank={false}
+                                            className="mt-2"
+                                            icon={
+                                                <ProfilePicture
+                                                    user={{
+                                                        hedgehog_config: {
+                                                            ...user?.hedgehog_config,
+                                                            use_as_profile: true,
+                                                        },
+                                                    }}
+                                                    size="sm"
+                                                    className="border bg-bg-light"
+                                                />
+                                            }
+                                        >
```

Comment:
Let's not use the same icon as the in-product Max, since this gives an extra false impression that this opens the in-product Max